packages/api/test/setup.ts

```diff
@@ -8,6 +8,8 @@ let server: FastifyInstance
 let client = new MongoClient(process.env.MONGO_URI)
 let db: Db
 
+jest.setTimeout(15000)
```

Comment:
what is this?

Reply:
I've added some test that need more than the default timeout (5 secs) to be finished. This line change the default jest timeout to 15 seconds